pages/projects/[id]/index.js

```diff
@@ -0,0 +1,106 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import { FaArrowLeftLong } from "react-icons/fa6";
+import { uid } from "uid";

```

Comment:
You don't use it anywhere so please remove it.

Reply:
We aren't sure what exactly you mean because we checked and we actually use all of these